Unable to install the project as an external contributor

The repository uses private npm packages:

```
npm ERR! code E401
npm ERR! 401 Unauthorized - GET https://npm.pkg.github.com/download/@apify-packages/ui-components/0.86.0/6927975c9bcef7f899f41292e4b3d2b675011690 - authentication token not provided
```

I'm filing this as an issue, because I think it would be vital of anyone could clone this repo and contribute to it. Even if only some components of the docs could run locally, in a similar sense like this:

https://github.com/apify/apify-docs/blob/82d6da569455bdc5a8e8dece260431959e04f5ef/CONTRIBUTING.md?plain=1#L83

Currently, it is not possible to even successfully finish `npm install` of the repo.

--- a/package.json
+++ b/package.json
@@ -56,7 +56,7 @@
         "typescript": "^5.1.3"
     },
     "dependencies": {
-        "@apify-packages/ui-components": "^0.86.0",
+        "@apify-packages/ui-components": "^0.108.0",
         "@apify/docsearch-apify-docs": "3.5.3",
         "@docusaurus/core": "^2.4.1",
         "@docusaurus/plugin-client-redirects": "^2.4.1",

